src/Networking/NexusMods.Networking.GOG/Client.cs

```diff
@@ -53,13 +55,15 @@ internal class Client : IClient
     /// <summary>
     /// TTL time for the secure URL cache.
     /// </summary>
-    private static readonly TimeSpan CacheTime = TimeSpan.FromMinutes(5);
+    private static readonly TimeSpan CacheTime = TimeSpan.FromSeconds(5);
```

Comment:
Just making sure, this change isn't a leftover of debug/testing, right?

Reply:
Yep, it was